content/sensu-go/5.18/installation/install-sensu.md

```diff
@@ -280,10 +273,12 @@ sensuctl configure -n \
 Here, the `-n` flag triggers non-interactive mode.
 Run `sensuctl config view` to see your user profile.
 
-We recommend that you change the default admin password immediately: `sensuctl user change-password --interactive`.
-
 For more information about sensuctl, see the [quickstart][23] and [reference][4] docs.
 
+### Change default admin password
+
+If you are using Docker and you did not use environment variables to override the default admin credentials in [step 2 of the backend installation process](#2-configure-and-start), we recommend that you change the default admin password immediately: `sensuctl user change-password --interactive`.
```

Comment:
Will they have completed initial sensuctl configuration at this point? I don't think they can change password without having run `sensuctl configure`

Reply:
Yes, I need to be more specific about that. I will update the instruction.